src/main/java/com/zufar/onlinestore/common/response/ApiResponse.java

```diff
@@ -0,0 +1,23 @@
+package com.zufar.onlinestore.common.response;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.LocalDateTime;
+
+public record ApiResponse<T>(
+
+        @JsonProperty("data")
+        T data,
+
+        @JsonProperty("message")
+        String message,
+
+        @JsonProperty("status")
+        Integer status,
+
+        @JsonProperty("time_stamp")
+        @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd HH:mm:ss")
```

Comment:
Please, create TIMESTAMP_JSON_FORMAT constant in `com.zufar.onlinestore.common.response;` and use it in this class.
```
public static final String TIMESTAMP_JSON_FORMAT = "yyyy-MM-dd HH:mm:ss";
```

Reply:
Done